src/argilla/server/elasticsearch.py

```diff
@@ -53,4 +54,19 @@ def _field_mapping_for_annotation(self, annotation_task: Annotation):
             # See https://www.elastic.co/guide/en/elasticsearch/reference/current/text.html
             return {"type": "text"}
         else:
-            raise ValueError(f"Annotation of type {settings_type} cannot be processed")
+            raise ValueError(f"ES mappings for Annotation of type {settings_type} cannot be generated")
+
+
+async def get_engine():
```

Comment:
Looks like we are using search engine everywhere. Maybe we should use `get_search_engine` instead:

```suggestion
async def get_search_engine():
```